include/SDL_float.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copying and distribution of this file, with or without modification,
+ * are permitted in any medium without royalty provided this notice is
+ * preserved.  This file is offered as-is, without any warranty.
+ * Names of contributors must not be used to endorse or promote products
+ * derived from this file without specific prior written permission.
+ */
+
+#ifndef SDL_float_h_
+#define SDL_float_h_
+
+#include <float.h>
+
+#define SDL_FLT_IS_NULL(value)                              \
+    ((value) > - FLT_EPSILON && (value) < + FLT_EPSILON)
+#define SDL_FLT_DIFF(value1, value2)            \
+    ((value1) - (value2))
+#define SDL_FLT_EQUAL(value1, value2)               \
```

Comment:
Suggest SDL_FLOAT_EQUAL
Also suggest SDL_FLOAT_EQUAL_EPSILON() where you can specify the epsilon value to use. There are a few places in SDL where we could take advantage of this.

Reply:
I propose `SDL_FLOAT_*` for the generic functions with epsilon and keep `FLT` and `DBL` (like libc and SDL) for the standard epsilon. If you are not ok, I will change of course.

Reply:
That sounds good.